frontend/src/composables/useNormRender.ts

```diff
@@ -7,12 +7,14 @@ import { UseFetchReturn } from "@vueuse/core"
  *
  * @param normXml XML of the norm that should be rendered
  * @param showMetadata Enable or disable metadata list at the beginning of the document
+ * @param snippet If the XML sent is only a snippet of a norm
  * @param customNorms The XMLs of norms which are referenced by the norm (e.g. in passiveModifications) and should be used instead of the data stored.
  * @param at Passive modifications coming into effect before this date should be applied before rendering the HTML
  */
 export function useNormRenderHtml(
   normXml: MaybeRefOrGetter<string | undefined>,
   showMetadata: MaybeRefOrGetter<boolean> = false,
+  snippet: MaybeRefOrGetter<boolean> = false,
   at?: MaybeRefOrGetter<Date | undefined>,
   customNorms?: MaybeRefOrGetter<string[] | undefined>,
 ): UseFetchReturn<string> {
```

Comment:
with that amount of optional parameters i would start thinking about moving them to an options object, so that the places calling this method only need to provide the parameters necessary (and it is less confusing which parameter is what _"was snippet the second or third parameter?"_) But no need to change it here now :)

Reply:
I understand. I will change it now